### ISSUE TITLE
No Centos builds with Ulauncher 5 

### ISSUE THREAD

**Issue by gornostal** [Ulauncher/Ulauncher#376]

**Comment by ababak** [comment]
Maybe, fall back to using https://pypi.org/project/pylev3/ if no binary is available?

**Comment by ababak** [comment]
It's pretty fast by the way

**Comment by gornostal** [comment]
@ababak we can't use a library from pypi because dependencies are installed listed in setup.cfg and are downloaded from centos repositories

**Comment by ababak** [comment]
It’s available from pip3 as well

**Comment by gornostal** [comment]
I understand. What I meant is I can only specify dependencies that exists in the main Centos repo.

**Comment by ababak** [comment]
Can it be used as a git submodule?

**Comment by gornostal** [comment]
No. I cannot see how git submodules could help with the problem.

If you really want to run Ulauncher in Centos, you could download the [latest release](https://github.com/Ulauncher/Ulauncher/releases/download/5.3.0-beta1/ulauncher_5.3.0.beta1.tar.gz) and then repeat these steps until it finally works

* run it (`./bin/ulauncher`)
* see import error
* fix the dependency issue by either installing it from Centos repos or pip3
* repeat

If you get it to work, I'd appreciate if you could post a comment here describing how you got the dependencies. That may be helpful for other Centos users

**Comment by ababak** [comment]
Yeah, it works. Thank you.

Here are the steps:
```bash
sudo yum install python36-dbus
sudo yum install python36-pyxdg
sudo yum install python3-devel
sudo pip3 install python-Levenshtein
sudo pip3 install pyinotify
./bin/ulauncher
```

There's one more warning to fix in the code:

ulauncher/ulauncher/main.py:8: PyGIWarning: Gtk was imported without specifying a version first. Use `gi.require_version('Gtk', '3.0')` before import to ensure that the right version gets loaded.
  `from gi.repository import Gtk`
